lib/store_base_sti_class_for_5_0.rb

```diff
@@ -346,6 +346,24 @@ def build_through_record(record)
     end
 
     module Reflection
+      class PolymorphicReflection < ThroughReflection
```

Comment:
In Rails `5.0.1` this class inherits from `AbstractReflection` , I'd check if all works with that change